app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -1,8 +1,9 @@
 import React, { useState } from "react";
 
+import { Check } from "@bigbinary/neeto-icons";
```

Comment:
Go through the Slack thread.

Reply:
Sorry, Yedhin this must be an old occurrence of the "@bigbinary". That has already been corrected. All the @bigbinary occurrences were removed.

Reply:
I am not sure what's going wrong. But the one place where you could definitely improve is your self-review process. Refer: https://handbook.neetokb.com/articles/git-and-github-essentials#self-review-process

Lemme know if you need any help regarding the same.

Reply:
Yes Yedhin I would be more cautious next time. But what I meant was that all such occurrences were eventually corrected in one of the future PRs. So as of the time you posted this review, I did not have anything to correct.